server/src/main/java/org/opensearch/index/store/remote/filecache/FileCacheStats.java

```diff
@@ -32,7 +33,7 @@
  *
  * @opensearch.api
  */
-@ExperimentalApi
+@PublicApi(since = "2.7.0")
```

Comment:
The annotation was flipped improperly before from @PublicApi(since = "2.7.0") to @ExperimentalApi